sdk/core/azure-core-http-okhttp/src/main/java/com/azure/core/http/okhttp/OkHttpAsyncHttpClient.java

```diff
@@ -169,7 +169,10 @@ public void onFailure(okhttp3.Call call, IOException e) {
 
         @Override
         public void onResponse(okhttp3.Call call, okhttp3.Response response) {
+            HttpResponse azureResponse = new OkHttpResponse(response, request);
+
             sink.success(new OkHttpResponse(response, request));
```

Comment:
Should this be using `azureResponse` now instead of creating another response instance inline?

Reply:
Yes, this should have returned `azureResponse`.